packages/react-celo/src/components/tray.tsx

```diff
@@ -173,8 +173,13 @@ export default function Tray({
               </span>
             </div>
           )}
-          {providers.map(([priority, providers]) => (
-            <div key={priority} className={styles.container}>
+          {providers.map(([priority, providers], i) => (
+            <div
+              key={priority}
```

Comment:
this isnt new but key should be unique

also `providers` is both what is being maped and the named variable in the map 

should it be `provider` the second time?

Reply:
provider.name would be a great key

Reply:
oh oops def an oversight

Reply:
still `providers` is both what is being mapped and the named variable in the map, im suprised this isnt causing a lint error. 

Can we rename one of them? The outter seems like its more providersByPriority ?

Reply:
Very true, I didn't notice the variable shadowing. Should definitely look into the eslint config here

Reply:
Added a new eslint rule for it.